java/src/main/java/ai/rapids/cudf/ColumnFilterOptions.java

```diff
@@ -53,12 +54,24 @@ public T includeColumn(String... names) {
       return (T) this;
     }
 
+    /**
+     * Include this column.
+     * @param name the name of the column
+     * @param isBinary whether this column is to be read in as binary
+     */
+    public T includeColumn(String name, boolean isBinary) {
+      includeColumnNames.add(name);
+      return (T) this;
+    }
+
```

Comment:
I'm wondering if we should just add this new overload in ParquetOptions - instead of adding this to the base class?  Do we expect that all input formats will support reading binary columns?